Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml

```diff
@@ -116,4 +116,16 @@
         acts: [ "Destruction" ]
   - type: Construction
     graph: CP14WallWood
-    node: WallCardboard
\ No newline at end of file
+    node: WallCardboard
+    
+- type: entity
+  id: CP14WallCyan
+  name: cyan wall
```

Comment:
maybe smtn like polished "marble" or "granite"

Reply:
Later, there will be separate granite walls

Reply:
Well, anyway, just the cyan wall is a weird name.

Reply:
It's more like spackle

Reply:
I have a bad fantasy(